resource-managers/nomad/src/main/scala/org/apache/spark/scheduler/cluster/nomad/ExecutorTask.scala

```diff
@@ -75,7 +75,7 @@ private[spark] object ExecutorTask
 
     task.addEnv("SPARK_EXECUTOR_MEMORY", jvmMemory(conf, task))
 
-    task.addEnv("SPARK_LOCAL_DIRS", "${NOMAD_ALLOC_DIR}")
+    task.addEnv("SPARK_LOCAL_DIRS", "/shuffle")
```

Comment:
curious, Why cant it stay as the default value `/tmp` ?

Reply:
I doubled checked and unfortunately I don't think we can isolate the shuffle data.  We could rename this to `/spark-local` or something similar though (I'd want to differentiate this from `/spark` which is already where we're putting our spark distribution)

Reply:
`/spark-work-dir` may be ?